src/main/java/mate/academy/dao/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exception.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+    private static final String USER_NAME = "root";
+    private static final String PASSWORD = "bodyaA20041@";
```

Comment:
Storing credentials in the source code is a security risk. Consider using environment variables or a configuration file to manage sensitive information securely.